crates/bevy_audio/src/sinks.rs

```diff
@@ -71,6 +71,10 @@ pub trait AudioSinkPlayback {
 /// Use [`AudioBundle`][crate::AudioBundle] to trigger that to happen.
 ///
 /// You can use this component to modify the playback settings while the audio is playing.
+///
+/// If this component is removed from an entity, and an [`AudioSource`][crate::AudioSource] is
+/// attached to that entity, that [`AudioSource`][crate::AudioSource] will start playing. If
+/// that source is unchanged, that translates to the audio restarting.
 #[derive(Component)]
 pub struct AudioSink {
```

Comment:
Does that also happen with `SpatialAudioSink`?

Reply:
yes!

Reply:
`SpatialAudioSink` has been addressed :+1: